Automattic/newspack-plugin
# Update the Primary Color to use Newspack new blue: `#36f`

### All Submissions:






### Changes proposed in this Pull Request:



* Update the main variables (`$primary_color`, `$primary_color_lighter`, `$primary_color_darker`)
* Adjust the Newspack logo
* Tweak the completed checklist icon to use blue
* Style the `is-default` button

### How to test the changes in this Pull Request:

Go through the setup and all the different screens of the plugin. Do you see our new blue?

### Other information:
